electron/utils.ts

```diff
@@ -549,6 +557,44 @@ function quoteIfNecessary(stringToQuote: string) {
   return stringToQuote
 }
 
+/**
+ * Checks if we're (1) on a Steam Deck and (2) running in Gaming Mode
+ */
+function isSteamDeckInGamingMode(): boolean {
+  if (!isLinux) {
+    return false
+  }
+
+  // Credits for this implementation go to https://gitlab.com/parallel-launcher/parallel-launcher
+
+  // Check if we're on a Deck by reading out /etc/os-release
+  let osReleasePath = '/etc/os-release'
+  // Just in case someone is *not* running the Flatpak
+  if (isFlatpak) {
+    osReleasePath = '/run/host' + osReleasePath
+  }
+
+  const fileContent = readFileSync(osReleasePath).toString()
+  const foundDeckVariant = fileContent.split('\n').find((line: string) => {
+    const [key, value] = line.split('=')
+    return key === 'VARIANT_ID' && value === 'steamdeck'
+  })
```

Comment:
As longer i think about we could go even cleaner

```suggestion
  const fileContent = readFileSync(osReleasePath).toString()
  const foundDeckVariant = fileContent.split('\n').find((line: string) => {
    return line.replaceAll(' ', '').includes('VARIANT_ID=steamdeck'')
  })
```